javascript/code-challenges/challenges-02.test.js

```diff
@@ -0,0 +1,317 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+
+CHALLENGE 1 - Review
+
+Write a function named raisedToTheThird that takes in an array of numbers and returns a new array of each of those numbers raised to the 3rd power (hint: look up Math.pow()). Use forEach to solve this problem.
+
+------------------------------------------------------------------------------------------------ */
+
+const raisedToTheThird = (arr) => {
+  const returnArray = [];
+  arr.forEach(num => {
+    returnArray.push(Math.pow(num, 3));
+
```

Comment:
I like that you're trying to semantically name your variables, however returnArray is a little clunky. I like "solution" or "answer".
Also, to save space, you have your forEach loop execute on one line!
```js
arr.forEach(num => solution.push(Math.power(num, 3));
```